src/inputs/Textarea.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+    <div class="textarea">
+        <label>{{ this.label }}</label>
+        <textarea v-on:input="e => this.$emit('onChange', this.name, e.target.value)" 
+        :value="this.value"
+        :id="this.id"
+        :rows="this.rows"
+        :cols="this.cols"
+        :disabled="this.disabled"
+        :name="this.name"
+        :readonly="this.readonly"></textarea> 
+    </div>
```

Comment:
the outer div is not needed here. You can make the `label` the outer most element, and put the textarea inside it.